crates/transaction-pool/src/pool/mod.rs

```diff
@@ -876,20 +896,6 @@ where
         self.pool.read().is_exceeded()
     }
 
-    /// Enforces the size limits of pool and returns the discarded transactions if violated.
-    ///
-    /// If some of the transactions are blob transactions, they are also removed from the blob
-    /// store.
-    pub(crate) fn discard_worst(&self) -> HashSet<TxHash> {
-        let discarded = self.pool.write().discard_worst();
-
-        // delete any blobs associated with discarded blob transactions
-        self.delete_discarded_blobs(discarded.iter());
```

Comment:
this logic has been removed now, right?

Reply:
It's moved to the only call site (`add_transactions`) now, in the `if !discarded.is_empty()` check.